tools/functional-tester/etcd-tester/main.go

```diff
@@ -49,6 +51,8 @@ func main() {
 	consistencyCheck := flag.Bool("consistency-check", true, "true to check consistency (revision, hash)")
 	isV2Only := flag.Bool("v2-only", false, "'true' to run V2 only tester.")
 	stresserType := flag.String("stresser", "default", "specify stresser (\"default\" or \"nop\").")
+	failureTypes := flag.String("failures", "default,failpoints", "specify failures (concat of \"default\" and \"failpoints\").")
+	terminate := flag.Bool("terminate", true, "'true' to terminate an etcd cluster after testing.")
```

Comment:
I'm assuming a workflow like below:

1. run `etcd-tester` with `--limit=1 --terminate=false` just for executing single round and keeping the cluster. The cluster spawned by `etcd-tester` is fuzzed by namazu. namazu captures packets between nodes by netfilter and increases randomness of the execution.
2. stop namazu and check the healthy of the cluster with `etcd-checker`
3. kill the cluster and goto 1

At first I thought activating namazu in the first step and deactivating in the second step by `etcd-tester` process would be simple. But I found that just executing namazu as an independent process from `etcd-tester` is possible and it will be simpler than the first idea (RPCs from `etcd-tester` isn't required at all). Is it acceptable?